library/ssl_tls13_keys.c

```diff
@@ -871,19 +871,24 @@ int mbedtls_ssl_tls1_3_generate_resumption_master_secret(
     int ret = 0;
 
     mbedtls_md_type_t md_type;
+    
+#if defined(MBEDTLS_DEBUG_C)  
     mbedtls_md_info_t const *md_info;
     size_t md_size;
-
+#endif
```

Comment:
```
/* MBEDTLS_DEBUG_C */
```
Curious, is there a reason that this was not added?

Reply:
LGTM

Reply:
```suggestion
#endif /* MBEDTLS_DEBUG_C */
```